tests/integrations/raftstore/test_conf_change.rs

```diff
@@ -254,6 +255,14 @@ fn test_server_pd_conf_change() {
 }
 
 fn wait_till_reach_count(pd_client: Arc<TestPdClient>, region_id: u64, c: usize) {
+    wait_till_reach_count_err(pd_client, region_id, c).unwrap();
+}
+
+fn wait_till_reach_count_err(
+    pd_client: Arc<TestPdClient>,
+    region_id: u64,
+    c: usize,
+) -> result::Result<(), String> {
```

Comment:
You can use `std::Result<...>` directly.
But IMO use a `String` as an Error type is not a good choice.